pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -19,32 +19,18 @@ package version
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/minishift/minishift/pkg/util"
 	"io"
 	"net/http"
+	"regexp"
 	"sort"
 	"strings"
-	"time"
+
+	"github.com/minishift/minishift/pkg/util"
 )
 
 type ImageTags struct {
-	Count    int         `json:"count"`
-	Next     string      `json:"next"`
-	Previous interface{} `json:"previous"`
-	Results  []Results   `json:"results"`
-}
-
-type Results struct {
-	Name        string      `json:"name"`
-	FullSize    int         `json:"full_size"`
-	Images      []ImageInfo `json:"images"`
-	ID          int         `json:"id"`
-	Repository  int         `json:"repository"`
-	Creator     int         `json:"creator"`
-	LastUpdater int         `json:"last_updater"`
-	LastUpdated time.Time   `json:"last_updated"`
-	ImageID     interface{} `json:"image_id"`
-	V2          bool        `json:"v2"`
+	Name  string `json:"name"`
+	Layer string `json:"layer"`
```

Comment:
No ImageTags. One does not have to map all fields of the JSON into the struct. You can map what you need. Other fields will be ignored.